src/olympia/templates/impala/base.html

```diff
@@ -24,9 +24,6 @@
     {% block site_css %}
       {{ css('zamboni/impala') }}
       {{ css('restyle/css') }}
-      <!--[if lt IE 9]>
-        <script src="{{ static('js/lib/iepp.min.js') }}"></script>
-      <![endif]-->
       <!--[if lte IE 9]>
         <link rel="stylesheet" href="{{ static('css/legacy/ie9.css') }}">
```

Comment:
According to GA, overall IE usage is low, as expected. IE 11 is the highest by far (28K sessions), and 8, 9, and 10 are roughly on par (700 - 1K sessions).

So, I think IE 7 and older can be dropped with very little loss. IE 10 and older can also be dropped, if needed.